pymongo/change_stream.py

```diff
@@ -55,7 +53,14 @@
 ])
 
 
-class ChangeStream(object):
+if TYPE_CHECKING:
+    from pymongo.client_session import ClientSession
+    from pymongo.collection import Collection
+    from pymongo.database import Database
+    from pymongo.mongo_client import MongoClient
+
+
+class ChangeStream(Generic[_DocumentType], Iterator[Mapping[str, Any]]):
```

Comment:
Does adding Iterator here do anything? IIRC mypy uses structural subtyping where we don't need to explicitly say that ChangeStream is an Iterator, mypy infers it from that fact that it implements the Iterator methods.

Reply:
Let's remove it